### PR TITLE
sia-ui: init at 1.4.3

### PR DESCRIPTION
**Pull request by 0xbe7a**
###### Motivation for this change
Sia is the "leading decentralized cloud storage platform" and the wallet was missing from nixpkgs
###### Things done

- [X] Tested using sandboxing ([nix.useSandbox](http://nixos.org/nixos/manual/options.html#opt-nix.useSandbox) on NixOS, or option `sandbox` in [`nix.conf`](http://nixos.org/nix/manual/#sec-conf-file) on non-NixOS linux)
- Built on platform(s)
   - [X] NixOS
   - [ ] macOS
   - [ ] other Linux distributions
- [ ] Tested via one or more NixOS test(s) if existing and applicable for the change (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
- [ ] Tested compilation of all pkgs that depend on this change using `nix-shell -p nixpkgs-review --run "nixpkgs-review wip"`
- [ ] Tested execution of all binary files (usually in `./result/bin/`)
- [ ] Determined the impact on package closure size (by running `nix path-info -S` before and after)
- [X] Ensured that relevant documentation is up to date
- [X] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/.github/CONTRIBUTING.md).

### CODE REVIEW

File: pkgs/applications/blockchains/sia-ui/default.nix [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+    install -m 444 -D ${appimageContents}/sia-ui.desktop $out/share/applications/sia-ui.desktop
+    install -m 444 -D ${appimageContents}/sia-ui.png $out/share/icons/hicolor/512x512/apps/sia-ui.png
+    substituteInPlace $out/share/applications/sia-ui.desktop \
+      --replace 'Exec=AppRun' 'Exec=${pname}'
+  '';
+
+  meta = with stdenv.lib; {
+    description = "A electron webapp to manage and interface with the Sia daemon";
+    homepage = "https://sia.tech/";
+    license = licenses.mit;
```
Comment (doronbehar) [inline pkgs/applications/blockchains/sia-ui/default.nix:39]: How come this is mit ? I couldn't find source code..

**Review by doronbehar — CHANGES_REQUESTED** [review]
Hey and sorry for this to have been waiting so long, I hope you are still interested.

your PR should have eventually two commits:

```
maintainers: add <me>
<package-name>: init at <version>
```

File: pkgs/applications/blockchains/sia-ui/default.nix (hunk 1 above)
Comment (samuelgrf) [reply pkgs/applications/blockchains/sia-ui/default.nix:39]: The source code is here: https://gitlab.com/NebulousLabs/Sia-UI

Comment (doronbehar) [reply pkgs/applications/blockchains/sia-ui/default.nix:39]: Would it be too much to ask this to be build from source? Considering this is an application that deals with crypto currency...

Comment (samuelgrf) [reply pkgs/applications/blockchains/sia-ui/default.nix:39]: I think any application that deals with cryptography or even money in this case should be compiled from source if possible.

**Comment by SuperSandro2000** [comment]
@0xbe7a please address the reviewer feedback.

**Comment by 0xbe7a** [comment]
> @0xbe7a please address the reviewer feedback.

Sorry for the late response, but i partly moved on from NixOS and Sia aswell. While i absolutely share your critique and this should be build by source, i no longer have interest in packaging this. Looking at the timeline of this pull request, i think its fair to say the community isn't interested either, so I'll close this.

**Pull request closed without merging**